tests/tox.sh

```diff
@@ -36,11 +36,15 @@ fi
 # copy the files to the root for the
 # types of images we're going to build
 mkdir -p {base,daemon,demo}
-# starting with kraken, the base image does not exist
 cp -Lrv ceph-releases/$CEPH_RELEASE/$IMAGE_DISTRO/base/* base || true
 cp -Lrv ceph-releases/$CEPH_RELEASE/$IMAGE_DISTRO/daemon/* daemon
 cp -Lrv ceph-releases/$CEPH_RELEASE/$IMAGE_DISTRO/demo/* demo || true
 
+# remove the base dir any release after jewel 
```

Comment:
ok, I can try. the testing script is also creating these directories though.

https://github.com/ceph/ceph-docker/blob/master/tests/tox.sh#L38